lib/experimental/class-gutenberg-rest-comment-controller.php

```diff
@@ -261,6 +271,211 @@ public function create_item_permissions_check( $request ) {
 		return true;
 	}
 
+	/**
+	 * Creates a comment.
+	 *
+	 * @since 4.7.0
+	 *
+	 * @param WP_REST_Request $request Full details about the request.
+	 * @return WP_REST_Response|WP_Error Response object on success, or error object on failure.
+	 */
+	public function create_item( $request ) {
+		// This code is copied exactly from (core file name) except for sectioned marked with the comment.
+		// '// Note: This is only relevant change for the backport.'
```

Comment:
```suggestion
		// This code is copied exactly from (core file name) except for sectioned marked with the comment.
```

Nit: I think the code here is the same as the core.

Reply:
The last part is an example of comment markings.